src/regions.py

```diff
@@ -0,0 +1,438 @@
+from collections import namedtuple
```

Comment:
can you add a comment on the intention of this file. e.g. all the APIs to interact with the regions table.

Reply:
I added a comment describing the intended usage of the regions table.